draft-gpew-priv-ppm.md

```diff
@@ -673,10 +683,7 @@ information specific to the extension.
 
 ### Report Storage
 
-The leader is required to buffer reports while waiting to aggregate them. The
-leader SHOULD NOT accept reports whose timestamps are too far in the future.
-Implementors MAY provide for some small leeway, usually no more than a few
-minutes, to account for clock skew.
+The leader is required to buffer reports while waiting to aggregate them.
```

Comment:
Optional: this is a really tiny section -- would this sentence fit somewhere else, allowing this section to be dropped? Perhaps in the description of the `/upload` flow.